core/src/script/CGXP/plugins/WFSPermalink.js

```diff
@@ -227,6 +236,10 @@ cgxp.plugins.WFSPermalink = Ext.extend(gxp.plugins.Tool, {
 
         var layerName = state.layer;
         delete state.layer;
+        if (state.show) {
+            this.showFeatures = false;
```

Comment:
Perhaps "state.show" is not the good parameter name to use? It seems to have side effects...
What about using "state.showFeatures" instead?